src/calculateRentalCost.js

```diff
@@ -28,7 +28,26 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_PER_DAY = 40;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM_DISCOUNT = 50;
+  const LONG_TERM = 7;
+  const SHORT_TERM = 3;
+  const rentSum = RENT_PER_DAY * days;
+
+  if (days >= LONG_TERM) {
+    return rentSum - LONG_TERM_DISCOUNT;
+  }
+
+  if (days >= SHORT_TERM && days < LONG_TERM) {
+    return rentSum - SHORT_TERM_DISCOUNT;
+  }
+
+  if (days < SHORT_TERM) {
+    return rentSum;
+  }
+
+  return rentSum;
```

Comment:
```suggestion
  return rentSum;
```
You don't need the last check as in both cases you return the same value